dockerfiles/theia/docker/ubi8/runtime-install-dependencies.dockerfile

```diff
@@ -8,5 +8,6 @@ USER root
 # Install curl and bash
 # Install ssh for cloning ssh-repositories
 # Install less for handling git diff properly
-RUN yum install -y sudo git bzip2 which bash curl openssh less && \
+# Install sshpass for handling passwordds for SSH keys
+RUN yum install -y sudo git bzip2 which bash curl openssh less sshpass && \
```

Comment:
is `sshpass` package present in the software repositories and it can be installed?

Reply:
It can be installed. I've build the image locally and the `sshpass` tool was present.

Reply:
I guess the same should be added downstream
https://github.com/redhat-developer/codeready-workspaces-theia/blob/master/conf/theia/ubi8-brew/runtime-install-dependencies.dockerfile#L11
@vinokurig check it with Nick, please
cc @nickboldt

Reply:
Yes but I'm not sure if we can use EPEL in downstream. 

And I can't build Theia from 7.13.x right now due to https://github.com/eclipse/che/issues/16951 so please don't commit this change until we have a working build process again.

Reply:
changed to build from sources